test-integration/test_client_covid_symptoms.py

```diff
@@ -3,7 +3,7 @@
 import json
 import unittest
 import ctakesclient
-from .test_resources import LoadResource
+from test.test_resources import LoadResource
```

Comment:
This is tough - it means that you can only run these tests if the toplevel src dir is in your PYTHONPATH. Which it is if you are running pytest from the toplevel dir, so it's mostly OK. But importing from non-installed modules feels slightly brittle.

I think a future re-org of this testing code and corpus is probably gonna happen, so it's fine for now. Just noting.

Reply:
Nevermind, I've changed my opinion on this - pytest is always run at toplevel anyway, and this guards against moving files around.